swagger/swagger.yaml

```diff
@@ -1906,6 +1902,13 @@ components:
         - SEARCH
         - SITREP
         - TAG_CHARACTER
+    BloodOxygenEnum:
+      type: string
+      description: The percentage of oxygen in someone's blood, as measured by a pulse oximeter
```

Comment:
Although blood ox is a percentage, this enum really isn't.  How about "A description of the percentage of oxygen in someone's blood, [...]" 
For similar vitals (heart rate and breathing) we just said, e.g., "Descriptive breathing level".  But I think "Descriptive blood oxygen level" is a bit austere...but I'm fine with it if you prefer the consistency with other vitals.

Reply:
I agree that the description could be made a bit clearer. Other than the few comments made, everything else seems to look fine to me.

Reply:
@Anchita14   Ayal has updated this PR.  If everything still looks fine to you, can you approve it, and take a look at the [accompanying client PR](https://github.com/NextCenturyCorporation/itm-evaluation-client/pull/44) ?  Thanks!